### DISCUSSION TITLE
Model popup is not working

### DISCUSSION THREAD

**Discussion by pokaleajit** [twbs/bootstrap#39327] [Q&A]
### Describe the issue

model and many features is not working after linking link in index.html
pls check screenshot 

### Reduced test cases

![Screenshot (1)](https://github.com/twbs/bootstrap/assets/148343433/f67b01ed-1257-4bba-88d3-2767aeadeaef)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v5.3

**Comment by julien-deramond** [comment]
@pokaleajit I've transformed your issue into a discussion because it lacks of detail. Please try to share information about your source code, your environment, and if possible via a reproducible environment like StackBlitz, CodePen, etc. Otherwise, it would be really difficult to help.

**Comment by adydetra** [comment]
@pokaleajit Please post your `index.html` code here. Example:

```html
is my index.html code
```
